handlers/api.go

```diff
@@ -3507,7 +3507,7 @@ func ApiWithdrawalCredentialsValidators(w http.ResponseWriter, r *http.Request)
 	vars := mux.Vars(r)
 	q := r.URL.Query()
 
-	credentialsOrAddressString := vars["withdrawalCredentialsOrEth1address"]
+	credentialsOrAddressString := ReplaceEnsNameWithAddress(vars["withdrawalCredentialsOrEth1address"])
 	credentialsOrAddressString = strings.ToLower(credentialsOrAddressString)
```

Comment:
This does not work. I have used `clarissa.eth` which is a valid ENS name with an address in our database: `0x000002b4A2e3459331b1eB6066A8FdB39c7fdC5D`
The `0x` then is truncated in `func ReplaceEnsNameWithAddress`: https://github.com/gobitfly/eth2-beaconchain-explorer/blob/65d4644759aa627e3907134a1d9971556e8cd72a/handlers/ens.go#L92-L100
And then `func IsValidEth1Address` says that this is not a valid address (as the prefix is missing).